crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/tests/handler_builder.rs

```diff
@@ -267,6 +267,7 @@ pub fn calculate_missing_ops(
 /// Test that the above functions work as expected in one specific case:
 /// Out of 6 ops total, all 3 agents hold 3 ops each.
 #[tokio::test(flavor = "multi_thread")]
+// @maackle Can we delete this?
 #[ignore = "This test doesn't make sense anymore because it's using the event sender as if there were separate databases"]
```

Comment:
Yes delete, and probably the function this is testing, and everything attached to it, can be deleted as well, since local sync is no longer a thing to be tested. But that can be followup. Maybe I'll add that to this PR?